dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -51,6 +51,8 @@ repertoire.verifyHostname=Host name verification
 repertoire.verifyHostname.desc=Specifies whether host name verification for outbound connections using a specific SSL configuration is enabled. If set to true, then all outbound SSL connections that use the specified SSL configuration undergo verification of the target server host name against that server's certificate. The attribute is set to false by default.
 repertoire.trustDefaultCerts=Trust the default certificates
 repertoire.trustDefaultCerts.desc=Specifies whether the trust manager can establish trust by using the default certificates. If set to true, then the default certificates are used in addition to the configured truststore file to establish trust. The attribute is set to false by default.
+repertoire.enforceCipherOrder=Enforce cipher order on server side
+repertoire.enforceCipherOrder.desc=Specified whether to indicate on the server socket to enforce the cipher order.  If set to true, the server socket will be enabled to enforce cipher order.  The attribute is set to false by default.                   
```

Comment:
Change,
"Specified whether to indicate on the server socket to enforce the cipher order.  If set to true, the server socket will be enabled to enforce cipher order.  The attribute is set to false by default."
to,
"Specify on the server socket whether to enforce the cipher order. If set to true, the server socket is enabled to enforce the cipher order. The attribute is set to false by default."

Reply:
Thank you for reviewing.  I've made the suggested update.

Reply:
Thanks. Looks good.